content/blog/explore-the-hpe-greenlake-for-compute-ops-management-rest-api-using-curl-and-postman.md

```diff
@@ -0,0 +1,333 @@
+
+The API supports several query parameters depending on the resource type, such as filter, limit (maximum number of records to return), offset (resource offset to start the response from) and sort (order in which to return the resources in the collection).
+
+Filters provide the ability to limit the resources that take part in the action of a REST call. When a REST call includes a filter, the GET or DELETE action is restricted to a response that meets the filter requirements. Filters are specified by using the query parameter 'filter'.
+
+Here is a simple example of filtering:
+
+**GET <URI>?filter=powerState eq 'Off'**
+
+This example shows a simple filter. The resources returned by the query are limited to results with the attribute **powerState** and the value **Off**.
+
+To use a filter on a nested property name, the '**/**' separator can be specified as follows:
+
+**GET /compute-ops/v1beta2/servers?filter=hardware/model eq 'ProLiant DL365 Gen10 Plus'\*\***\
+{\*\*
+
+    **"offset": 0,**
+
+    **"count": 20,**
+
+    **"total": 20,**
+
+    **"items": [**
+
+        **{**
+
+            **"id": "P39368-B21+CN70421C51",**
+
+            **"type": "compute-ops/server",**
+
+            **"platformFamily": "PROLIANT",**
+
+            **"resourceUri": "/compute-ops/v1beta2/servers/P39368-B21+CN70421C51",**
+
+            **"name": "HPE-HOL56",**
+
+            **"createdAt": "2022-04-29T12:35:35.265978+00:00",**
+
+            **"updatedAt": "2022-10-25T19:54:36.572565+00:00",**
+
+            **"generation": 292,**
+
+            **"hardware": {**
+
+                **"serialNumber": "CN70421C51",**
+
+                **"model": "ProLiant DL365 Gen10 Plus",**
+
+                **"uuid": "33393350-3836-4E43-3730-343231433531",**
+
+                **"productId": "P39368-B21",**
+
+                **"powerState": "ON",**
+
+                **"indicatorLed": "OFF",**
+
+                **"health": {**
+
+                 **   "summary": "OK",**
+
+                 **   "healthLED": "OK",**
+
+                 **   "fans": "OK",**
+
+                 **   "fanRedundancy": "REDUNDANT",**
+
+                 **   "liquidCooling": "NOT_PRESENT",**
+
+                 **   "liquidCoolingRedundancy": "NOT_PRESENT",**
+
+                 **   "memory": "OK",**
+
+                 **   "network": "UNKNOWN",**
+
+                 **   "powerSupplies": "OK",**
+
+                 **   "powerSupplyRedundancy": "NOT_PRESENT",**
+
+                 **   "processor": "OK",**
+
+                 **   "storage": "OK",**
+
+                 **   "temperature": "OK",**
+
+                 **   "bios": "OK",**
+
+                 **   "smartStorage": "OK"**
+
+                **},**
+
+                **"bmc": {**
+
+                 **   "mac": "B4:7A:F1:4E:9E:92",**
+
+                 **   "ip": "172.30.231.116",**
+
+                 **   "hostname": "None"**
+
+The following cURL command includes the filter
```

Comment:
Add a colon after "filter" so it reads "... includes the filter:" (because you said "The following..)